stack_test.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from stack import Stack
+
+def test_stack_push():
+    my_stack = Stack([1, 2, 3])
+    my_stack.push('hello')
+    assert my_stack._storage._size == 4
```

Comment:
`._size` is a private method of your LinkedList, you shouldn't be accessing it from anywhere outside your linked list.

Instead use `assert len(my_stack)`